charts/karmada-dashboard/values.yaml

```diff
@@ -19,20 +19,14 @@ global:
 ## See https://yaml.org/spec/1.2/spec.html#id2765878 and
 ## https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_advanced_syntax.html#yaml-anchors-and-aliases-sharing-variable-values
 ## for more details about yaml anchors and aliases.
-karmadaDashboardImageVersion: &karmadaDashboardImageVersion latest
+karmadaDashboardImageVersion: &karmadaDashboardImageVersion main
 
 ## Specify the imagePullPolicy of karmada-dashboard images
 ## Defaults to 'IfNotPresent', you can set to 'Always' in cases of special needs
 karmadaDashboardImagePullPolicy: &karmadaDashboardImagePullPolicy IfNotPresent
 
 podDisruptionBudget: &podDisruptionBudget { }
 
-## @param installMode "host" and "agent" are provided
-## "host" means install karmada in the control-cluster
-## "agent" means install agent client in the member cluster
-## "component" means install selected components in the control-cluster
-installMode: "host"
```

Comment:
This is completely unnecessary; the dashboard currently doesn't require this kind of config.